packages/gatsby-theme-aio/src/components/GetCredential/Organization.js

```diff
@@ -0,0 +1,272 @@
+                            margin : 0 ;
+                          }
+
+                          & > div > .spectrum-Picker-popover > ul > li > div > div > svg {
+                            @media screen and (min-width:${MIN_MOBILE_WIDTH}) and (max-width:${MAX_TABLET_SCREEN_WIDTH}){
+                              margin: 3px;
+                              padding: 0;
+                            }
+                          }
+
+                            padding: 5px;
+                            border-radius: 3px;
+                            border: 1px solid #D0D0D0 !important;
+
+                          ` }
+                      >
+                        <Picker
+                          isQuiet
+                          items={allOrganization?.map((organs, k) => {
+                            return {
+                              title: organs?.name,
+                              selected: k === selectedIndex
+                            }
+                          })}
+                          onChange={(index) => {
+                            setSelectedIndex(index);
+                          }}
+                        />
+                      </div>
+
+                    </div>
+                    <div
+                      css={css`
+                        display: flex;
+                        gap: 5px;
+                      `}
+                    >
+                      <span>Can't find your organization?</span>
+                      <a href="https://some_help_link"
+                        target="_blank"
+                        rel="noreferrer"
+                        css={css`
+                          color:rgb(0, 84, 182);
+                          &:hover {
+                            color: rgb(2, 101, 220);
+                          }`
+                        }
+                      >Learn more about organizations.</a>
+                    </div>
+
+                    <div className="spectrum-ButtonGroup spectrum-Dialog-buttonGroup spectrum-Dialog-buttonGroup--noFooter" css={css` gap: 20px; `} >
+                      <button className="spectrum-Button spectrum-Button--sizeM spectrum-Button--outline spectrum-Button--secondary spectrum-ButtonGroup-item" type="button" onClick={() => { handleClick("cancel") }}>
+                        <span className="spectrum-Button-label">Cancel</span>
+                      </button>
+                      <button className="spectrum-Button spectrum-Button--sizeM spectrum-Button--fill spectrum-Button--accent spectrum-ButtonGroup-item" type="button" onClick={() => { handleClick("save") }}>
+                        <span className="spectrum-Button-label" >Save</span>
+                      </button>
+                    </div>
+                  </div>
+
+                </section>
+
+              </div>
+              :
+              <Loading />
+            }
+          </div>
+        </div>

```

Comment:
Instead of writing pure HTML code here, can we use existing spectrum components? We should do this everywhere if not already.